src/components/modals/ActionScorer.tsx

```diff
@@ -95,7 +95,7 @@ function getColumns(intl: InjectedIntl): IRenderableColumn[] {
                     return (
                         <OF.PrimaryButton
                             data-testid="action-scorer-button-clickable"
-                            onClick={() => component.handleActionSelection(action.actionId)}
+                            onClick={() => component.handleActionSelection(action.actionId, action)}
```

Comment:
Why not just pass action?  Of is there a case where the actionId is coming from a different action?

Reply:
Yea, talked about that in the description:

> There was TODO about making the handleActionSelection function take in an action, we currently do it for SET_ENTITY actions since it was required, but left the other actions as if to minimize changes. It's probably good time to take that task on soon to simplify code.

I can't remember exactly but I think there where the id that it was given came form either ScoredAction or Action and this mean we couldn't always pass in the same thing and would hve to do the search for action anyways, and didn't want to risk the change yet but it's very confusing and would also like to simplify.